### ISSUE TITLE
[Bug]: Accessibility: low contrast in light mode

### ISSUE THREAD

**Issue by shortwavesurfer2009** [meshtastic/Meshtastic-Android#1704]
### What happened?

With such a light green bar, the buttons should really be black instead of white.

![Image](https://github.com/user-attachments/assets/abce72c5-d221-4df9-8cab-cc7002a47487)
![Image](https://github.com/user-attachments/assets/4be9fa46-4347-49fb-adad-1c631967d3b5)

### App Version

2.5.19

### Phone

Oneplus Nord n200

### Device

Na

### Firmware

Na

### Relevant log output

```shell

```

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by shortwavesurfer2009** [comment]
Actually, I just updated to the newest version and it's worse because now the navigation icons such as messages, contacts, etc. are white on the green background as well.
